docs/advanced-topics/network-policies.asciidoc

```diff
@@ -0,0 +1,427 @@
+:page_id: network-policies
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+:api_server_port: 443
+:apm_port: 8200
+:dns_port: 53
+:ent_port: 3002
+:es_http_port: 9200
+:es_transport_port: 9300
+:kb_port: 5601
+:webhook_port: 9443
+
+
+[id="{p}-{page_id}"]
+= Network policies
+
+link:https://kubernetes.io/docs/concepts/services-networking/network-policies/[Network policies] allow you to isolate pods by restricting incoming and outgoing network connections to a trusted set of sources and destinations. This section describes how to use network policies to isolate the ECK operator and the {stack} applications to a set of namespaces to implement a form of soft multi-tenancy. Soft multi-tenancy is a term used to describe a scenario where a group of trusted users (different teams within an organization, for example) share a single resource such as a Kubernetes cluster. Note that network policies alone are not sufficient for security. You should complement them with strict RBAC policies, resource quotas, node taints, and other available security mechanisms to ensure that tenants cannot access, modify, or disrupt resources belonging to each other.
```

Comment:
How about adding the term and relevant definition of `Soft multi-tenancy` to our [Glossary](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-glossary.html)?

Reply:
The term is used only once in this single document and the definition follows immediately after it was used.  It is not used anywhere else in the documentation. Since it is not a common term, is it worth including in the glossary?